### PR TITLE
Querying ownerCollection from firebase

### PR DESCRIPTION
**Pull request by MarcusBoay**
I made many many changes do how the query should be

### CODE REVIEW

File: app/src/main/java/com/example/atheneum/fragments/OwnerPageFragment.java [hunk 1]
```diff
@@ -31,10 +56,40 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
...
+    /**
+     * Retrieve books from Firebase.
+     *
+     * See: https://stackoverflow.com/questions/37902635/no-setter-field-for-warning-firebase-database-retrieve-data-populate-listview
+     */
+    public void retrieveBooks() {
+        FirebaseUser firebaseUser = FirebaseAuth.getInstance().getCurrentUser();
+        final FirebaseDatabase db = FirebaseDatabase.getInstance();
+        String dbOwnerBooksPath = getString(R.string.db_ownerCollection) + "/" + firebaseUser.getUid().toString();
+        DatabaseReference ref = db.getReference(dbOwnerBooksPath);
```
Comment (ishchow) [inline app/src/main/java/com/example/atheneum/fragments/OwnerPageFragment.java]: Try getting the reference by using: `
```
DatabaseRef ref = db.getReference.child(getString(R.string.db_ownerCollection)).child(firebaseUser.getUid());
```

I've never had to convert firebaseUser.getUid() to a string to query firebase so maybe that was what the problem was.

File: app/src/main/java/com/example/atheneum/fragments/OwnerPageFragment.java [hunk 2]
```diff
@@ -31,10 +56,40 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
...
+
+        if (firebaseUser != null) {
+            ref.addValueEventListener(new ValueEventListener() {
```
Comment (ishchow) [inline app/src/main/java/com/example/atheneum/fragments/OwnerPageFragment.java]: You might need to use a ChildEventListener the values under ownerCollection aren't books but they're keys so you have to listen for changes to those keys I think?

https://stackoverflow.com/questions/37902635/no-setter-field-for-warning-firebase-database-retrieve-data-populate-listview

Comment (ishchow) [reply app/src/main/java/com/example/atheneum/fragments/OwnerPageFragment.java]: Even the code snippet you were basing it off of used child event listeners:

https://firebase.google.com/docs/database/admin/retrieve-data

Comment (ishchow) [reply app/src/main/java/com/example/atheneum/fragments/OwnerPageFragment.java]: This is probably the real issue

**Comment by MarcusBoay** [comment]
This pull request contains change regarding owner books.

Main things:
Retrieving owner books from firebase via manual parsing (TODO: parse owner and borrower field in book, make firebase parser classes)
Made RecyclerView for list of owner books and show all books by owner (TODO: make each book selectable to go to inidividual book activity)
Updated strings.xml to contain more database fields

**Comment by ishchow** [comment]
@MarcusBoay Looks okay to me but man what is up with your branch? It's really hard to read your PR since other peoples' commits ended up on it.

**Comment by MarcusBoay** [comment]
Might be because I rebased onto master a bunch?

**Pull request merged**